### ISSUE TITLE
Locale setup problems

### ISSUE THREAD

**Issue by kor-bim** [Blarc/ai-commits-intellij-plugin#39]
I'm a South Korean
Even though I changed Locale to Korea during the setup, the message generation phrase comes out only in English

**Comment by lyrl** [comment]
I also have the same problem.

**Comment by Blarc** [comment]
What kind of prompt do you use? It needs to contain something similar to `the response must be in the language {locale}`. The `{locale}` string is used as a variable and gets replaced with the locale that is set in the settings.

I've tried generating a commit message in Korean with the following prompt and it worked:
```
Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything, the response must be in the language {locale} and must NOT be longer than 74 characters. The sent text will be the differences between files, where deleted lines are prefixed with a single minus sign and added lines are prefixed with a single plus sign.
{diff}
```

**Comment by lyrl** [comment]
This is the prompt I am using.
```
You are to act as the author of a commit message in git. Your mission is to create clean and comprehensive commit messages in the conventional commit convention and explain why a change was done. I'll send you an output of 'git diff --staged' command, and you convert it into a commit message.Use GitMoji convention to preface the commit.Don't add any descriptions to the commit, only commit message.
Use the present tense. Lines must not be longer than 74 characters. Use {locale} to answer. {diff}
```


![image](https://user-images.githubusercontent.com/1203738/235072372-dcfa435b-d9e6-4b77-865d-7a757f7e2d6a.png)
![image](https://user-images.githubusercontent.com/1203738/235072478-d2070ef2-eacf-4908-9ee4-1d0ff2c11d50.png)

**Comment by Blarc** [comment]
It seems that if locale `Chinese (Simplified)` is used, the model doesn't understand the locale correctly, because prompt replaces `{locale}` variable with `Chinese (Simplified)` instead of `Chinese`. I replaced `locale.displayName` with `locale.displayLanguage` which shows only basic languages.

This will be released with the next version **1.1.0**. Till then, you try to use `Chinese` instead of `Chinese (Simplified)`.

**Comment by Blarc** [comment]
Version **1.1.0** that contains the fix was released.
